src/OpenTelemetry.Exporter.OpenTelemetryProtocol/OtlpExporterOptionsExtensions.cs

```diff
@@ -42,7 +48,38 @@ public static Channel CreateChannel(this OtlpExporterOptions options)
             }
 
 #if NETSTANDARD2_1 || NET6_0_OR_GREATER
-            return GrpcChannel.ForAddress(options.Endpoint);
+            var retryPolicy = new RetryPolicy
+            {
+                MaxAttempts = RetryMaxAttempts,
+                InitialBackoff = RetryInitialBackoff,
+                MaxBackoff = RetryMaxBackoff,
+                BackoffMultiplier = RetryBackoffMultiplier,
+
+                // See table showing gRPC codes and if they are retryable:
+                // https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/protocol/otlp.md#failures
+                RetryableStatusCodes =
+                {
+                    StatusCode.Cancelled,
+                    StatusCode.DeadlineExceeded,
+                    StatusCode.ResourceExhausted, // TODO: Investigate this one.
```

Comment:
looks like we need to follow https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/protocol/otlp.md#otlpgrpc-throttling for this one.

Reply:
My initial assumption is that the gRPC client handles this. Can probably validate the behavior conforms with the specs requirements with the mock server.

Reply:
looks like it does https://github.com/grpc/grpc-dotnet/blob/23047e0d498501dc95bb58b010ed3c11f8cb3cf8/src/Grpc.Net.Client/Internal/Retry/RetryCall.cs#L190
if I am looking at right place 😄